src/index.html

```diff
@@ -5,9 +5,34 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="product" data-qa="card">
+      <img
+      class="product__img" src="./images/imac.jpeg"
+      alt="imac"
+      >
+      <h2 class="product__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
```

Comment:
H1-H6 tags are meant for section's heading. IMO `p` tag would fit better here. [Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements)